### PR TITLE
Add pulses and sources to Zebra device

### PR DESCRIPTION
**Pull request by noemifrisina**
Adds a few source and some PVs to the Zebra, needed for running serial on I24.

Needed for [MX_BLEUSKY#10](https://github.com/DiamondLightSource/mx_bluesky/issues/10)

### Instructions to reviewer on how to test:
1. Run tests

### Checks for reviewer
- [ ] Would the PR title make sense to a scientist on a set of release notes

### CODE REVIEW

**Review by DominicOram — APPROVED** [review]
Minor comment, take it or leave it

File: src/dodal/devices/zebra.py [hunk 1]
```diff
@@ -108,6 +112,11 @@ def is_armed(self) -> bool:
 
 class ZebraOutputPanel(Device):
     pulse_1_input = epics_signal_put_wait("PULSE1_INP")
+    pulse_1_delay = epics_signal_put_wait("PULSE1_DLY")
+    pulse_1_width = epics_signal_put_wait("PULSE1_WID")
+    pulse_2_input = epics_signal_put_wait("PULSE2_INP")
+    pulse_2_delay = epics_signal_put_wait("PULSE2_DLY")
+    pulse_2_width = epics_signal_put_wait("PULSE2_WID")
```
[the remark this answers is not in the document]
Comment (noemifrisina) [reply src/dodal/devices/zebra.py]: Decided to take it, should make things a bit tidier.

File: src/dodal/devices/zebra.py [hunk 2]
```diff
@@ -107,8 +117,15 @@ def is_armed(self) -> bool:
         return self.arm.armed.get() == 1
 
 
+class PulseOutput(Device):
+    pulse_inp = epics_signal_put_wait("_INP")
+    pulse_dly = epics_signal_put_wait("_DLY")
+    pulse_wid = epics_signal_put_wait("_WID")
```
Comment (DominicOram) [inline src/dodal/devices/zebra.py:123]: Nit: The word pulse here is redundant, they're in the pulse class

**Review by DominicOram — DISMISSED** [review]
Thanks, I think the `PulseOutput` is cleaner. We will need to update Hyperion to use it though, can you add a PR there please?

**Review by dperl-dls — APPROVED** [review]
Thanks!

**Pull request merged**